### PR TITLE
Exporter mapping improvements

### PR DESCRIPTION
**Pull request by trask**
A few things:
* by adding special case for "spring-scheduling" INTERNAL span, it will prevent misc INTERNAL spans from showing up in request table
* fixed mapping of CONSUMER spans (due to new instrumentation in 0.10.0 that hasn't been released yet)
* fixed dependency type for messaging spans that are not PRODUCER

### CODE REVIEW

File: agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java [hunk 1]
```diff
@@ -118,16 +119,19 @@ private void export(SpanData span) {
...
+            } else if ("spring-scheduling".equals(stdComponent) && !SpanId.isValid(span.getParentSpanId())) {
+                // TODO need semantic convention for determining whether to map INTERNAL to request or dependency
+                //  (or need clarification to use SERVER for this)
                 exportRequest(span, attributes);
             } else {
                 exportRemoteDependency(span, attributes, true);
             }
         } else if (kind == Kind.CLIENT || kind == Kind.PRODUCER) {
             exportRemoteDependency(span, attributes, false);
+        } else if (kind == Kind.CONSUMER && !span.getHasRemoteParent()) {
```
Comment (heyams) [inline agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java:131]: getHasRemoteParent, can we fix it in the opentelemetry-api? it doesn't seem to follow the naming convention for boolean getter.

File: agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java [hunk 2]
```diff
@@ -118,16 +119,19 @@ private void export(SpanData span) {
             boolean isLog = removeAttributeBoolean(attributes, AI_INTERNAL_LOG);
             if (isLog) {
                 exportLogSpan(span, attributes);
-            } else if (!SpanId.isValid(span.getParentSpanId())) {
-                // TODO revisit this decision
-                // maybe user-generated telemetry?
-                // otherwise this top-level span won't show up in Performance blade
+            } else if ("spring-scheduling".equals(stdComponent) && !SpanId.isValid(span.getParentSpanId())) {
```
Comment (heyams) [inline agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java:122]: Should i rename SpringScheduler to SpringScheduling for smoke test?  It's SpringSchedulingInstrumentation on the opentelemetry as well.

**Review by heyams — APPROVED** [review]
👍

File: agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java (hunk 2 above)
Comment (trask) [reply agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java:122]: sure, that seems more consistent 👍

File: agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java (hunk 1 above)
Comment (trask) [reply agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java:131]: good point! I just opened https://github.com/open-telemetry/opentelemetry-java/issues/1910

**Pull request merged**